webui/react/src/hooks/useModal/Project/useModalProjectCreate.tsx

```diff
@@ -51,7 +51,11 @@ const useModalProjectCreate = ({ onClose, workspaceId }: Props): ModalHooks => {
 
   const handleOk = useCallback(async () => {
     try {
-      const response = await createProject({ description, name, workspaceId });
+      const response = await createProject({
+        description: description.trim(),
```

Comment:
if we always want this, it could be something that `createProject` does.

Reply:
or better probably a backend treatment but that might take longer

Reply:
Backend would be nice. One can still create Workspaces/Projects with leading or trailing whitespace via the CLI, but you'd really have to go out of your way to do so.